bin/express-cli.js

```diff
@@ -281,8 +281,9 @@ const db = mongojs(dbUri, collections);
         case 'sequelize':
           pkg.dependencies['mysql2'] = '^1.6.4'
           pkg.dependencies['sequelize'] = '^4.41.2'
+          pkg.dependencies['dotenv'] = '^8.2.0'
           app.locals.localModules.db = './models'
-          www.locals.db = `
+          app.locals.db = `
```

Comment:
need to also remove the logic from `bin/www.js.ejs` thats using www.locals.db

Reply:
ah gotcha